stytch/stytchapi/stytchapi.go

```diff
@@ -44,30 +45,40 @@ func WithLogger(logger Logger) Option {
 	return func(api *API) { api.logger = logger }
 }
 
+// WithBaseURI overrides the client base URI determined by the environment.
+//
+// The value derived from stytch.EnvLive or stytch.EnvTest is already correct for production use
+// in the live or test environment, respectively. This is implemented to make it easier to use
+// this client to access development versions of the API.
```

Comment:
```suggestion
// this client to access internal development versions of the API.
```
Small clarification.